pycbc/workflow/grb_utils.py

```diff
@@ -457,10 +457,9 @@ def build_veto_filelist(workflow):
     return veto_files
 
 
-def build_segment_filelist(workflow):
+def build_segment_filelist(seg_dir):
     """Construct a FileList instance containing all segments txt files"""
 
-    seg_dir = workflow.cp.get('workflow', 'segment-dir')
     file_names = ["bufferSeg.txt", "offSourceSeg.txt", "onSourceSeg.txt"]
```

Comment:
A suggestion here to combine these 3 files into one (these were horrible even in 2011).

Reply:
... Probably not for this PR though.

Reply:
I opened an issue https://github.com/gwastro/pycbc/issues/4887 for future PyGRB development.  Thanks!